lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -205,7 +205,7 @@ function gatherTapTargets(tapTargetsSelector, className) {
 
   /** @type {{
     tapTargetElement: Element,
-    clientRects: ClientRect[]
+    clientRects: LH.Artifacts.Rect[]
```

Comment:
these haven't been real `ClientRect`s in a long time. Not sure why this wasn't previously an error.